shanoir-ng-users/.classpath

```diff
@@ -15,6 +15,7 @@
 		<attributes>
 			<attribute name="optional" value="true"/>
 			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="test" value="true"/>
```

Comment:
Can you explain ?

Reply:
I removed this, it was probably added by eclipse or JAVA_LANGUAGE_SERVER...